src/main/resources/db/migration/V091__breathing_space_emails.sql

```diff
@@ -0,0 +1,3122 @@
+			      DEFAULT,
+			      'Breathing space notifications',
+			      'Hysbysiadau lle anadlu',
+			      'bath.breathingspace@justice.gov.uk',
+			      '',
+			      ''
+		      )
+		RETURNING id into temp_id;
+
+		INSERT INTO public.search_courtemail(id, court_id, email_id, "order")
+		VALUES(
+			      DEFAULT,
+			      (SELECT id FROM public.search_court WHERE slug = 'bath-law-courts-civil-family-and-magistrates'),
+			      temp_id,
+			      3
+		      );
+	END
+$$;
+
+DO $$
+	DECLARE temp_id integer;
+	BEGIN
+		INSERT INTO public.search_email(id, description, description_cy, address, explanation, explanation_cy)
+		VALUES(
+			      DEFAULT,
+			      'Breathing space notifications',
+			      'Hysbysiadau lle anadlu',
+			      'luton.breathingspace@justice.gov.uk',
+			      '',
+			      ''
+		      )
+		RETURNING id into temp_id;
+
+		INSERT INTO public.search_courtemail(id, court_id, email_id, "order")
+		VALUES(
+			      DEFAULT,
+			      (SELECT id FROM public.search_court WHERE slug = 'bedford-county-court-and-family-court'),
+			      temp_id,
+			      3
+		      );
+	END
+$$;
+
+DO $$
+	DECLARE temp_id integer;
+	BEGIN
+		INSERT INTO public.search_email(id, description, description_cy, address, explanation, explanation_cy)
+		VALUES(
+			      DEFAULT,
+			      'Breathing space notifications',
+			      'Hysbysiadau lle anadlu',
+			      'birkenhead.breathingspace@justice.gov.uk',
+			      '',
+			      ''
+		      )
+		RETURNING id into temp_id;
+
+		INSERT INTO public.search_courtemail(id, court_id, email_id, "order")
+		VALUES(
+			      DEFAULT,
+			      (SELECT id FROM public.search_court WHERE slug = 'birkenhead-county-court'),
+			      temp_id,
+			      3
+		      );
+	END
+$$;
+
+DO $$
+	DECLARE temp_id integer;
+	BEGIN
+		INSERT INTO public.search_email(id, description, description_cy, address, explanation, explanation_cy)
+		VALUES(
+			      DEFAULT,
+			      'Breathing space notifications',
+			      'Hysbysiadau lle anadlu',
+			      'birmingham.breathingspace@justice.gov.uk',
+			      '',
+			      ''
+		      )
+		RETURNING id into temp_id;
+
+		INSERT INTO public.search_courtemail(id, court_id, email_id, "order")
+		VALUES(
+			      DEFAULT,
+			      (SELECT id FROM public.search_court WHERE slug = 'birmingham-civil-and-family-justice-centre'),
+			      temp_id,
+			      3
+		      );
+	END
+$$;
+
+DO $$
+	DECLARE temp_id integer;
+	BEGIN
+		INSERT INTO public.search_email(id, description, description_cy, address, explanation, explanation_cy)
+		VALUES(
+			      DEFAULT,
+			      'Breathing space notifications',
+			      'Hysbysiadau lle anadlu',
+			      'blackpool.breathingspace@justice.gov.uk',
+			      '',
+			      ''
+		      )
+		RETURNING id into temp_id;
+
+		INSERT INTO public.search_courtemail(id, court_id, email_id, "order")
+		VALUES(
+			      DEFAULT,
+			      (SELECT id FROM public.search_court WHERE slug = 'blackpool-family-court'),
+			      temp_id,
+			      3
```

Comment:
Does the order need to go up by one each time?

Reply:
nah, I'm just inserting it as a the 3rd link in the list.